python_modules/dagster/dagster/core/asset_defs/asset_selection.py

```diff
@@ -19,26 +19,33 @@ def all() -> "AllAssetSelection":
         return AllAssetSelection()
 
     @staticmethod
-    def keys(*key_strs: str) -> "KeysAssetSelection":
-        return KeysAssetSelection(*key_strs)
+    def keys(*key_coercables: CoerceableToAssetKey) -> "KeysAssetSelection":
```

Comment:
Elsewhere, when we have args of type `CoerceableToAssetKey`, we just call them `key` or `asset_key` - doing that here would be more consistent and avoid introducing a term that users are otherwise unfamiliar with.